modules/ppcp-settings/resources/js/Components/Screens/Onboarding/Components/PaymentFlow.js

```diff
@@ -42,7 +42,10 @@ const PaymentFlow = ( {
 			/>
 		);
 	}
-
+	let description = optionalDescription;
+	if ( ! useAcdc ) {
+		description = '';
+	}
```

Comment:
I think we can simplify this to:

```suggestion
const description = useAcdc ? optionalDescription : '';
```